assets/scss/_home.scss

```diff
@@ -170,6 +170,29 @@
             animation: reel 4s linear infinite;
         }
     }
+    #complextopics .container {
+        margin-top: auto;
+    }
+    .workshop-container {
+        position: relative;
+        width: 100%;
+    }
+    .workshop {
+        position: relative;
+        width: 100%;
+        height: 60vw;
+        min-height: 400px;
+        max-height: 643px;
+        background-image: url(../img/home/border-bottom.png), url(../img/home/Workshop_Background.png);
+        background-size: 25px, cover;
+        background-repeat: repeat-x, no-repeat;
+        background-position: bottom,top;
```

Comment:
It looks like the submarine is supposed to be visible at the mobile size, so changing this to `background-position: bottom, 20%;` should put it closer to the right spot. This could be set at a media query for `$break-sm-xs`  but I also think it works ok at 20% for all sizes up to the 1600px layout change below - up to you.